<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctlPage</class>
 <widget class="QWidget" name="ctlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>774</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>774</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>989</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <widget class="QSlider" name="horizontalSliderDq">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxSpeedP">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxFps">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>位置</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTarPos">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lableData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>data：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>帧率</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>力矩</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SPEED P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SPEED I</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCal">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>标定</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMinSpeed">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-5000</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSync">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>sync</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>POS D</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMaxSpeed">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-5000</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>POS P</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxSpeedI">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxCur">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-50.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPosP">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lableFps">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>实时帧率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>速度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MAX</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMinCur">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-50.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDqP">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderSpeed">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DQ P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>90</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MIN</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDqI">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MAX</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPosD">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>50</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DQ I</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MIN</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="WidgetPlot2D" name="widgetPlot2D" native="true">
     <property name="minimumSize">
      <size>
       <width>650</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPlot2D</class>
   <extends>QWidget</extends>
   <header location="global">widgetplot2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
